amato/umbrel-app.yml

```diff
@@ -0,0 +1,21 @@
+manifestVersion: 1
+id: amato
+category: productivity
+name: Amato Timer
+version: "0.0.1.1"
+tagline: Simple work timer.
+description: >-
+  A simple timer with default 25 minute increments to track your work.
```

Comment:
It looks like it's not really 25 minute "increments" since there are 3 set options of 5 minutes, 10 minutes, and 25 minutes with no ability to increment higher:
![image](https://github.com/getumbrel/umbrel-apps/assets/85373263/365bab4f-2d73-4fce-8d0e-e651d1a0e2a1)

Instead, should we say something like:
`A simple timer to help track your time spent on tasks and boost your productivity. Choose between 5, 10, and 25 minute timers`